src/meltano/cli/state.py

```diff
@@ -310,12 +310,26 @@ def get_state(ctx: click.Context, project: Project, state_id: str) -> None:
 
 @meltano_state.command(cls=InstrumentedCmd, name="clear")
 @prompt_for_confirmation(prompt="This will clear state for the job. Continue?")
-@click.argument("state-id")
+@click.argument("state-id", required=False)
+@click.option(
+    "--all",
+    is_flag=True,
+    required=False,
+    help="Helps clear all states. Use with --force flag to skip confirmation.",
```

Comment:
question: let me know if you plan to implement this. I think it's certainly worth it since it's a destructive action, but should it be something like `--yes/-y` instead?

Reply:
Nevermind, this flag was already present.